actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -186,7 +185,7 @@
     private List<String> basePackages = new ArrayList<>();
 
     @Config("orbit.actors.pulseInterval")
-    private long pulseIntervalMillis = TimeUnit.SECONDS.toMillis(10);
+    private long pulseIntervalMillis = TimeUnit.SECONDS.toMillis(1);
```

Comment:
Is this intentional?  1ms pulse seems excessive

Reply:
It's a 1-second pulse, converted into milliseconds, not a 1ms pulse.

Reply:
right, I mean is 1 second excessive?  it was 10 seconds before, and does making it 1 introduce anything that we should be aware of?